src/os/Journal.h

```diff
@@ -71,6 +72,8 @@ class Journal {
 
   virtual bool should_commit_now() = 0;
 
+  virtual int _op_journal_transactions_prepare(list<ObjectStore::Transaction*>& tls, bufferlist& tbl) = 0;
```

Comment:
also, while we are here, let's make the tbl arg a bufferlist *tbl so that it's clear it's an output argument (most of this old code doesn't follow the coding style, but we should update it when it's convenient to do so)